addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1323,10 +1350,35 @@ MockServer.include({
      * @param {integer} message_id
      */
     _mockMailChannel_SetLastSeenMessage(ids, message_id) {
+        // deprecated: update mail.channel/fetched_message_id and mail.channel/seen_message_id
```

Comment:
mail.channel meant to be mail.channel.partner?

Reply:
It intentionally refers to `mail.channel` (fake) fields.

Reply:
In the end, I just removed the `mail.channel` fake fields `fetched_message_id` and `seen_message_id`.